extension/.nycrc.json

```diff
@@ -21,6 +21,6 @@
   "report-dir": ".coverage",
   "statements": 82,
   "branches": 64,
-  "functions": 84,
+  "functions": 80,
```

Comment:
I guess I'll have to trust you on the test coming later :stuck_out_tongue_winking_eye: